easybuild/easyconfigs/c/CoinUtils/CoinUtils-2.11.3-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = "ConfigureMake"
+
+name = 'CoinUtils'
+version = '2.11.3'
+
+homepage = "https://github.com/coin-or/CoinUtils"
+description = """CoinUtils (Coin-OR Utilities) is an open-source collection of classes and functions that are generally
+ useful to more than one COIN-OR project."""
+
+source_urls = ['https://www.coin-or.org/download/source/%(name)s/']
+sources = [SOURCE_TGZ]
+checksums = ['7c364792effe89d78b9b5385f30eaccc0fe92aab1caf5a1a835d81680639911f']
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('Doxygen', '1.8.14'),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['include', 'lib'],
```

Comment:
@tstrempel It's helpful to make this a bit more specific, I would use this:

```python
sanity_check_paths = {
    'files': ['libCoinUtils.%s' % SHLIB_EXT],
    'dirs': ['include/coin', 'lib/pkgconfig'],
}
```

Reply:
I changed it to this: 
```
sanity_check_paths = {
    'files': ['lib/libCoinUtils.%s' % SHLIB_EXT],
    'dirs': ['include/coin', 'lib/pkgconfig'],
}
```

Without the `lib/` it threw an error on my end, but all of the checks passed...

Reply:
Not including the `lib/` was my mistake of course.

The tests run in GitHub don't actually try to install the software, so they wouldn't know whether or not the sanity check paths are correct.

We do a test build on systems where we have sufficient resources to make sure the easyconfig file is correct, I'll look into this right now and get this merged, thanks a lot!